app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -275,6 +276,11 @@ class PatientSummaryScreen(
         .subscribe(::openContinuation)
   }
 
+  private fun teleconsultStatusChanged() = screenRouter.streamScreenResults()
+      .ofType<ActivityResult>()
+      .extractSuccessful(TELECONSULT_STATUS_SHEET) {}
```

Comment:
This extension is meant to extract a result of the result intent. If you only care whether it was successful and not what the result is, maybe a new extension method like `.filterIfSuccess()` is a better option here than adding a blank lambda?

Reply:
That's a good point, will add an extension.